guides/common/modules/proc_synchronizing-the-project-client-name-repository-for-rhel-7-and-rhel-6.adoc

```diff
@@ -12,6 +12,10 @@ To use the CLI instead of the {ProjectWebUI}, see the procedure relevant for you
 * xref:CLI_Synchronizing_the_Client_Repository_rhel_7_{context}[]
 * xref:CLI_Synchronizing_the_Client_Repository_rhel_6_{context}[]
 
+.Prerequisites
+* You have enabled the repository.
+For more information, see xref:enabling-the-project-client-name-repository-rhel-7-and-rhel-6[].
```

Comment:
I don't think you need this line because the procedure appears in the same assembly. I would spell out the name of the repository in the preceding line.